backend/src/akvo/flow_services/stats.clj

```diff
@@ -68,14 +71,30 @@
                      (conj data kinds)))))
 
 (defn get-all-data [server-list kinds]
-  (for [server server-list
-        :let [stats (get-stats server kinds)]
-        :when stats]
-    (conj (calc-stats kinds stats) server)))
+  (for [server server-list]
+    (e/if-ok (get-stats server kinds)
+             #(conj (calc-stats kinds %) server))))
+
+(defn too-many-errors? [all-data]
+  (let [errors (vec (filter e/error? all-data))]
+    (cond
+      (empty? errors) {:level :info :message "No errors collecting stats" :data ""}
+
+      (< 0.2
```

Comment:
Why this _magic_ number?

Reply:
We expect most of the flow instances to work, or what is the same, little to fail. More than 20% broken seems to me a strong indication that something is broken for real.